src/vs/editor/common/config/editorOptions.ts

```diff
@@ -1586,7 +1637,8 @@ export class EditorOptionsValidator {
 			wordWrapBreakBeforeCharacters: _string(opts.wordWrapBreakBeforeCharacters, defaults.wordWrapBreakBeforeCharacters),
 			wordWrapBreakAfterCharacters: _string(opts.wordWrapBreakAfterCharacters, defaults.wordWrapBreakAfterCharacters),
 			wordWrapBreakObtrusiveCharacters: _string(opts.wordWrapBreakObtrusiveCharacters, defaults.wordWrapBreakObtrusiveCharacters),
-			autoClosingBrackets: _boolean(opts.autoClosingBrackets, defaults.autoClosingBrackets),
+			autoClosingBrackets: _autoClosingOptions(opts.autoClosingBrackets, defaults.autoClosingBrackets),
+			autoClosingQuotes: _autoClosingOptions(opts.autoClosingQuotes, defaults.autoClosingQuotes),
```

Comment:
We may want to make sure the old boolean `autoClosingBrackets` contributes to `autoClosingQuotes` as well. If `autoClosingBrackets` is `true`, which is the old default value, `autoClosingQuotes` needs to be true otherwise users will miss auto quotes closing.